src/lib.rs

```diff
@@ -68,6 +68,49 @@ fn do_validate_annotations(pod: &apicore::Pod, settings: &settings::Settings) ->
     Ok(())
 }
 
+/* Function used to check if the seccomp profile type is defined in the
+ * allowed_profiles settings
+ */
+fn allowed_profiles_has_profile_type(
+    settings: &settings::Settings,
+    seccomp_profile: &apicore::SeccompProfile,
+) -> bool {
+    // Unfortunately, we cannot store the allowed_profiles settings parsed to
+    // avoid this iteration in every evaluation.
```

Comment:
I don't think this is going to a big performance issue, because I don't expect many security profiles to be specified. However, if that's the case, we can always build a small lookup table inside of the `validate` function. But I would definitely leave that for later